pkgs/applications/science/astronomy/galario/default.nix

```
@@ -0,0 +1,75 @@
+{ stdenv
+, fetchurl
+, cmake
+, fftw
+, fftwFloat
+, enablePython ? false
+, pythonPackages
+, llvmPackages
+}:
+
+stdenv.mkDerivation rec {
+  name = "galario-${version}";
+  version = "1.2.1";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/mtazzari/galario/archive/v${version}.tar.gz";
+      sha256 = "1m1lc6z5db6h2l719080jlh2aaaa82p1ddfzsv0kqlh6biyx6psq";
+    })
+    (fetchurl {
+      url = "https://github.com/UCL/GreatCMakeCookOff/archive/v2.1.9.tar.gz";
+      sha256 = "0b31q59x8kjhnnkdqryafkag55nd2y86lj4wac0wz58aigi24fz3";
+    })
+  ];
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ fftw fftwFloat ]
+    ++ stdenv.lib.optional enablePython pythonPackages.python
+    ++ stdenv.lib.optional stdenv.isDarwin llvmPackages.openmp
+  ;
+
+  propagatedBuildInputs = stdenv.lib.optional enablePython [
+    pythonPackages.numpy
+    pythonPackages.cython
+    pythonPackages.pytest
+  ];
+
+  checkInputs = stdenv.lib.optional enablePython pythonPackages.scipy;
+
+  sourceRoot = "${name}";
+
+  preConfigure = ''
+    mkdir -p build/external/src/
+    cp -r ../GreatCMakeCookOff-2.1.9 build/external/src/GreatCMakeCookOff
+  '';
+
+  preCheck = ''
+    ${if stdenv.isDarwin then "export DYLD_LIBRARY_PATH=$(pwd)/src/" else "export LD_LIBRARY_PATH=$(pwd)/src/"}
+    ${if enablePython then "sed -i -e 's|^#!.*|#!${stdenv.shell}|' python/py.test.sh" else ""}
+  '';
+
+  doCheck = true;
+
+  postInstall = stdenv.lib.optionalString (stdenv.isDarwin && enablePython) ''
+    install_name_tool -change libgalario.dylib $out/lib/libgalario.dylib $out/lib/python*/site-packages/galario/double/libcommon.so
+    install_name_tool -change libgalario_single.dylib $out/lib/libgalario_single.dylib $out/lib/python*/site-packages/galario/single/libcommon.so
+  '';
+
+  meta = with stdenv.lib; {
+    description = "GPU Accelerated Library for Analysing Radio Interferometer Observations";
+    longDescription = ''
+      Galario is a library that exploits the computing power of modern
+      graphic cards (GPUs) to accelerate the comparison of model
+      predictions to radio interferometer observations. Namely, it
+      speeds up the computation of the synthetic visibilities given a
+      model image (or an axisymmetric brightness profile) and their
+      comparison to the observations.
+    '';
+    homepage = https://mtazzari.github.io/galario/;
```

We're probably going to deprecate unquoted URLs if RFC 45 is approved.

```suggestion
    homepage = "https://mtazzari.github.io/galario/";
```